public/paths/connections/connections.yml

```diff
@@ -0,0 +1,95 @@
+get:
+  tags:
+    - Connections
+  summary: List connections
+  operationId: listConnection
+  description: Receive list of connections saved in the database.
+  parameters:
+  - in: query
+    name: from
+    description: Source entity, must be used with to and cannot be used with entity
```

Comment:
Similar here, what is the actual difference between from and entity? How can we guide an API user to choose the right parameter.